File: C/1006.c
#include <stdio.h>

int main()
{

    double n1, n2, n3, media;
    scanf("%lf%lf%lf", &n1, &n2, &n3);
    n1 = n1 * 2.0;
    n2 = n2 * 3.0;
    n3 = n3 * 5.0;
    media = (n1 + n2 + n3) / 10.0;
    printf("MEDIA = %.1lf\n", media);

    return 0;
}

File: GO/1006.go
package main

import "fmt"

func Media2(a, b, c float64) float64 {
	return (a*2.0 + b*3.0 + c*5.0) / 10.0
}
func main() {
	var a, b, c float64

	fmt.Scan(&a)
	fmt.Scan(&b)
	fmt.Scan(&c)

	fmt.Printf("MEDIA = %.5f\n", Media2(a, b, c))
}
